usaspending_api/accounts/urls_federal_obligations.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import url
+
+from usaspending_api.accounts.views import federal_obligations as views
+
+# bind ViewSets to URLs
+federal_list = views.FederalAccountByObligationViewSet.as_view(
+    {'get': 'list', 'post': 'list'})
+federal_detail = views.FederalAccountByObligationViewSet.as_view(
+    {'get': 'retrieve', 'post': 'retrieve'})
+
+urlpatterns = [
+    url(r'^$', federal_list),
+    url(r'(?P<pk>[0-9]+)/$', federal_detail)
```

Comment:
We have this route here to allow the PK to be passed in (`/api/v2/federal_obligations/:id`) but I didn't notice it being used in the `FederalAccountByObligationViewSet`. Is it still necessary to publish this route? I also didn't didn't see it in the associated gist.